### ISSUE TITLE
Exiting prematurely if `--tags wcag21aa` is present?

### ISSUE THREAD

**Issue by beep** [dequelabs/axe-cli#96]
This feels like it’s probably user error, but I’m unable to return any test results when `--tags wcag21aa` is present:

```
$ axe --version
3.0.0
$ axe cnn.com --tags wcag21aa
Running axe-core 3.2.2 in chrome-headless

Testing http://cnn.com ... please wait, this may take a minute.
axe-webdriverjs deprecated Error must be handled as the first argument of axe.analyze. See: #83 .nvm/versions/node/v12.2.0/lib/node_modules/axe-cli/lib/axe-test-urls.js:85:8
Failed to inject axe-core into one of the iframes!
  0 violations found!

Please note that only 20% to 50% of all accessibility issues can automatically be detected. 
Manual testing is always required. For more information see:
https://dequeuniversity.com/curriculum/courses/testingmethods

$
```

If I omit the flag, then things run as expected:

```$ axe cnn.com
Running axe-core 3.2.2 in chrome-headless

Testing http://cnn.com ... please wait, this may take a minute.
axe-webdriverjs deprecated Error must be handled as the first argument of axe.analyze. See: #83 .nvm/versions/node/v12.2.0/lib/node_modules/axe-cli/lib/axe-test-urls.js:85:8
Failed to inject axe-core into one of the iframes!

  Violation of "button-name" with 1 occurrences!

…

74 Accessibility issues detected.

Please note that only 20% to 50% of all accessibility issues can automatically be detected. 
Manual testing is always required. For more information see:
https://dequeuniversity.com/curriculum/courses/testingmethods

$
```

Is this a known issue? And/or am I doing something wrong? (I’m guessing it’s the latter 😅)

**Comment by straker** [comment]
The `--tags` option only runs axe-core rules that have the matching tag. Currently axe-core only has [2 wcag21aa rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md): `autocomplete-valid` and `css-orientation-lock`. Experimental rules aren't run by default, so only the `autocomplete-valid` rule is run, which passes.

**Comment by beep** [comment]
Ah, got it. Thanks very much for the context, @straker.

**Comment by martincomito** [comment]
Wich rules and tags runs by default if I don´t use the --tag flag?

**Comment by straker** [comment]
All [rules without the experimental tag](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) will be run.

**Comment by martincomito** [comment]
Thanks a lot @straker !
